frontend/src/common/entities.ts

```diff
@@ -75,7 +77,7 @@ export interface Author {
 
 export interface Collection {
   access_type: ACCESS_TYPE;
-  consortia: string[];
+  consortia: CONSORTIA[];
```

Comment:
@tihuan I apologise - the request for changing `consortia` from `string[]` to `CONSORTIA[]` was accidentally dropped from https://github.com/chanzuckerberg/single-cell-data-portal/pull/3875.